<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClass</class>
 <widget class="QWidget" name="AddClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加课程</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/image/windowIcon.png</normaloff>:/image/image/windowIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>312</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>科目名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="className"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>授课老师：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="teacherName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>教室    ：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="classRoom"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>上课时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>星期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="weekNum"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>上课节次     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="time"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>上课周期：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>从</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fromWeek"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>周  到</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="toWeek"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>周</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="singleWeek">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>单周</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="doubleWeek">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>双周</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="allWeek">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>全部</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/buttonBk.png);
color: rgb(255, 255, 255);
font: 75 14pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/image/buttonBk.png);
color: rgb(255, 255, 255);
font: 75 14pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="msg">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>请填写完整信息!</string>
   </property>
  </widget>
  <widget class="QLabel" name="msg_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 14pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>添加或修改课程：</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>confirm</zorder>
  <zorder>quit</zorder>
  <zorder>msg</zorder>
  <zorder>msg_2</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
